ocs_ci/ocs/ui/views.py

```diff
@@ -948,6 +948,16 @@
     "bucketclass-breadcrumb": ("//a[normalize-space()='BucketClasses']", By.XPATH),
 }
 
+OBC = {
```

Comment:
There is already obc locators. Is there a reason to add another one?
https://github.com/red-hat-storage/ocs-ci/blob/fac1441e69015368037bc70ab4eec2068c581a1b/ocs_ci/ocs/ui/views.py#L249-L267
Maybe you can integrate your locators into this one.

Reply:
No. I missed to check existing obc locator